authenticate.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+from getpass import getpass
+import logging
+from pathlib import Path
```

Comment:
Well spotted. It's weird because yes we _are_ testing 3.3 and I don't understand why they are passing.  My best guess it that `requests` is bringing in `pathlib` as a transitive dependency. Maybe we should just drop support for 3.3...

Reply:
...The alternative is to add it as a dependency, because it does work with 3.3 - it's just not bundled

Reply:
...Nope I dropped suport 😄